lib/sles4sap/sap_deployment_automation_framework/networking.pm

```diff
@@ -70,28 +71,74 @@ This leaves 16 IP addresses for each subnet. (only 14 are usable).
 sub calculate_subnets {
     my (%args) = @_;
     $args{network_space} //= calculate_net_addr_space();
-    my @subnet_names = qw(
-      admin_subnet_address_prefix
-      db_subnet_address_prefix
-      app_subnet_address_prefix
-      web_subnet_address_prefix
+    # extract subnet prefix from network addr space and calculate total IPs available for all subnets
+    my $network_space_ip_count = calculate_ip_count(subnet_prefix => (split(/\//, $args{network_space}))[1]);
+
+    # Definition of subnet name and subnet prefix
+    # Subnet assignment is defined here approximately how many IPs specific subnet might need
+    # total count of all subnet IPs must stay within network space boundaries
+    my %subnet_definition = (
+        'admin_subnet_address_prefix' => '/29',
+        'web_subnet_address_prefix' => '/29',
+        'db_subnet_address_prefix' => '/28',
+        'app_subnet_address_prefix' => '/28',
+        'iscsi_subnet_address_prefix' => '/28'
```

Comment:
All these `''` are unnecessary when using `=>`, as you did in `configure_tfvars.pm`.

Reply:
Oh yes, you are correct. Weird that github check didn't caught that.
Fixed.